models/public/anti-spoof-mn3/anti-spoof-mn3.md

```diff
@@ -0,0 +1,73 @@
+# anti-spoof-mn3
+
+## Use Case and High-Level Description
+
+The `anti-spoof-mn3` model is an anti-spoofing binary classificator based on the [MobileNetv3](https://arxiv.org/abs/1905.02244), trained on the [CelebA-Spoof dataset](https://arxiv.org/abs/2007.12342). It's a small, light model, trained to predict whether or not a spoof RGB image given to the input. A lot of advanced techniques have been tried and selected the best suit options for the task.
+For details see original [repository](https://github.com/kirillProkofiev/light-weight-face-anti-spoofing).
+
+## Example
```

Comment:
we are removing empty sections from documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please provide sample image or remove this section